pages/user_profile.py

```diff
@@ -0,0 +1,32 @@
+from methods import Methods
+from locators import Locators, LocatorsUsers
+from pages.home_page import HomePage
+
+class UserProfilePage(HomePage):
```

Comment:
Why inheritance starts from HomePage, guess it should be BasePage.

Reply:
Ok